### PR TITLE
Lg/0.6.9 release notes

### PR DESCRIPTION
**Pull request by lsgunnlsgunn**
**Overview**
Highlight some key updates in the 0.6.9 (and related) SDK release.

### CODE REVIEW

File: modules/release-notes/pages/sdk-release-notes.adoc [hunk 1]
```diff
@@ -48,19 +48,45 @@ dfx canister install -all
...
 ....
++
+Note that you specify the argument you want to pass using the Candid interface description format.
 
 * A new `+--memory allocation+` command-line option enables you to specify a memory allocation when deploying a canister.
 
 * The `+dfx identity new+` command has been enhanced to display a confirmation message when adding a new identify is successful.
 
-== Highlights of what's new in 0.6.7
+* Key updates for {proglang} (0.4.4) include backend support for the following features:
```
Comment (chenyan-dfinity) [inline modules/release-notes/pages/sdk-release-notes.adoc:65]: The version number is internal, and should not be exposed to user.
```suggestion
* Key updates for {proglang} include backend support for the following features:
```

File: modules/release-notes/pages/sdk-release-notes.adoc [hunk 2]
```diff
@@ -48,19 +48,45 @@ dfx canister install -all
...
+
+** Preliminary support for importing and exporting actor classes and for dynamic canister installation.
+** Unit installation arguments for actors.
+** Improved error handling for actor classes and input/output exceptions.
+** Improved garbage collection.
```
Comment (chenyan-dfinity) [inline modules/release-notes/pages/sdk-release-notes.adoc:70]:
```suggestion
** Preliminary support for importing and exporting actor classes and for dynamic canister installation.
** Improved error handling for actor classes and input/output exceptions.
```

File: modules/release-notes/pages/sdk-release-notes.adoc [hunk 3]
```diff
@@ -48,19 +48,45 @@ dfx canister install -all
 +
 For a preview of how to use the simplified developer workflow, see https://sdk.dfinity.org/docs/quickstart/dfx-deploy-quickstart.html[Developer Workflow Preview].
 
-* You can now specify arguments on the command-line when you deploy a single canister using the `+dfx canister install+` command.
+* You can now specify arguments on the command-line when you deploy a **single** canister using the `+dfx canister install+` command.
```
Comment (chenyan-dfinity) [inline modules/release-notes/pages/sdk-release-notes.adoc:51]:
```suggestion
* You can now specify arguments on the command-line when you deploy a **single** canister using actor class with the `+dfx canister install+` command.
```

File: modules/release-notes/pages/sdk-release-notes.adoc [hunk 4]
```diff
@@ -48,19 +48,43 @@ dfx canister install -all
...
+
+** Preliminary support for importing and exporting actor classes and for dynamic canister installation.
+** Improved error handling for actor classes and input/output exceptions.
+
+This release also includes the following BREAKING CHANGE: 
+
+* The Motoko compiler no longer supports variable or type declarations being specified before defining an actor or an actor class. 
+Only import declarations can be used as leading declarations for programs that define an actor or an actor class.
++
+The recommended workaround is to move the declarations that previously preceded the main actor to an imported module then use a call to the imported declaration within the main actor.
```
Comment (crusso) [inline modules/release-notes/pages/sdk-release-notes.adoc:75]:
```suggestion
The recommended workaround is either:
* to move the declarations that preceded the main actor into the actor's body, or
* to move the declarations into to a new, imported module, referenced from the main actor.
```

File: modules/release-notes/pages/sdk-release-notes.adoc [hunk 5]
```diff
@@ -48,19 +48,43 @@ dfx canister install -all
...
+For example, if the `+counterA+` is declared in the `+mylib.mo+` module, you might use code similar to the following:
++
+....
+import mylib "./mylib";
+
+actor Counter { 
+  mylib.counterA()
+  ...
+....
```
Comment (crusso) [inline modules/release-notes/pages/sdk-release-notes.adoc:84]:
```suggestion
For example:
+
....
type State = { #red; #orange; #green }
var state : State = #red;
actor TrafficLight { 
  ...
}
```
may be re-written as:
```
actor TrafficLight { 
  type State = { #red; #orange; #green };
  var state : State = #red;
  ...
}
```

File: modules/release-notes/pages/sdk-release-notes.adoc [hunk 6]
```diff
@@ -95,7 +119,7 @@ This new support for multiple identities enables you to test role-based access c
 ** Language support to enable canisters to take installation arguments.
```
Comment (crusso) [inline modules/release-notes/pages/sdk-release-notes.adoc:119]: A parameterized actor *class* defines a canister that takes an argument on installation.

Comment (crusso) [inline modules/release-notes/pages/sdk-release-notes.adoc:119]:
```suggestion
** Language support to enable canisters to take installation arguments. A parameterized actor  *class* defines a canister that takes an argument on installation.
```

File: modules/release-notes/pages/sdk-release-notes.adoc [hunk 7]
```diff
@@ -48,19 +48,43 @@ dfx canister install -all
...
 
-== Highlights of what's new in 0.6.7
+* Key updates for {proglang} include backend support for the following features:
+
+** Preliminary support for importing and exporting actor classes and for dynamic canister installation.
+** Improved error handling for actor classes and input/output exceptions.
+
+This release also includes the following BREAKING CHANGE: 
+
+* The Motoko compiler no longer supports variable or type declarations being specified before defining an actor or an actor class.
```
Comment (crusso) [inline modules/release-notes/pages/sdk-release-notes.adoc:72]:
```suggestion
* The Motoko compiler no longer supports arbitrary declarations preceeding the main actor or an actor class. 
```

File: modules/release-notes/pages/sdk-release-notes.adoc [hunk 8]
```diff
@@ -48,19 +48,43 @@ dfx canister install -all
...
+Note that you specify the argument you want to pass using the Candid interface description format.
 
 * A new `+--memory allocation+` command-line option enables you to specify a memory allocation when deploying a canister.
 
 * The `+dfx identity new+` command has been enhanced to display a confirmation message when adding a new identify is successful.
 
-== Highlights of what's new in 0.6.7
+* Key updates for {proglang} include backend support for the following features:
+
+** Preliminary support for importing and exporting actor classes and for dynamic canister installation.
```
Comment (crusso) [inline modules/release-notes/pages/sdk-release-notes.adoc:67]:
```suggestion
** Preliminary support for importing actor classes.
** Preliminary support for dynamic canister installation (by calling the constructor of an imported actor class).
```

File: modules/release-notes/pages/sdk-release-notes.adoc [hunk 9]
```diff
@@ -48,19 +48,43 @@ dfx canister install -all
...
+** Improved error handling for actor classes and input/output exceptions.
```
Comment (crusso) [inline modules/release-notes/pages/sdk-release-notes.adoc:68]: what does this refer to?

File: modules/release-notes/pages/sdk-release-notes.adoc (hunk 8 above)
Comment (lsgunnlsgunn) [reply modules/release-notes/pages/sdk-release-notes.adoc]: Here's where it came from ...
== 0.4.4 (2020-09-21)
= Motoko compiler changelog

== 0.4.4 (2020-09-21)
Actor class export
Accept unit installation args for actors
Reject platform actor (class) programs with additional decs
Handle IO exceptions at the top-level
RTS: Remove duplicate array and blob allocation code
RTS: Fix pointer arithmetic in BigInt collection function

== 0.4.3 (2020-09-14)

Preliminary support for actor class import and dynamic canister installation. Surface syntax may change in future.
BREAKING CHANGE: a compilation unit/file defining an actor or actor class may only have leading import declarations; other leading declarations (e.g. let or type) are no longer supported.
Rust GC

File: modules/release-notes/pages/sdk-release-notes.adoc (hunk 9 above)
Comment (lsgunnlsgunn) [reply modules/release-notes/pages/sdk-release-notes.adoc]: I was trying to generalize these items in a "positive" way:
Reject platform actor (class) programs with additional decs
Handle IO exceptions at the top-level

**Pull request merged**